pack.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
     - OpsGenie
     - alerting
     - monitoring
-version: 0.3.2
+version: 0.4.0
 author: Jon Middleton
 email: jon.middleton@pulsant.com
```

Comment:
While I'm doing a PR, is there another value I can put here. I no longer have this email address and passed on maintainership of this pack some time ago (see #9).

Reply:
That's fine. Please change it to:

```yaml
author: StackStorm, Inc.
email: info@stackstorm.com
```

Reply:
Cheers, updated and tagged the issue to be closed on merge.